backend-project/config/settings/base.py

```diff
@@ -123,7 +123,7 @@
 
 STATIC_URL = "/static/"
 STATIC_ROOT = "./static/"
```

Comment:
Tutaj można zrobić to samo.

Reply:
tzn STATIC_ROOT?
STATIC_URL chyba powinien pozostać bez zmian

Reply:
Tak